### ISSUE TITLE
Allow setting a time offset?

### ISSUE THREAD

**Issue by kartikynwa** [hadronzoo/theme-changer#29]
First of all thank you for writing and maintaining this project. I use it everyday.

Not sure how useful this will be generally but I would like the theme to change at one hour before sunset rather than exactly at sunset because it starts to get pretty dark at that point. I would appreciate allowing to set an offset for this.

If this is something worth pursuing, I can try implementing this if you can give me some pointers on how to get started. I have never worked on emacs plugins before but I am willing to try.

Thank you again.

**Comment by swflint** [comment]
Yes, I think that could be a helpful feature.

You'd want to start looking at lines 217--221, which are where the "next change time" is calculated.  You'd probably want the variable to include both a sunrise and sunset offset (car & cdr respectively?).

**Comment by kartikynwa** [comment]
This seems to already have been implemented.

https://github.com/hadronzoo/theme-changer/blob/7febd7632451bb99a5d92f24623432c4de035ff1/theme-changer.el#L97-L100

I can set it to `-3600` which is good enough for me. I have confirmed that it works too. I also don't need separate delays for sunrise/sunset because of my sleeping hours. If this kind of granularity is desired I can implement it. Otherwise feel free to close the issue.
